scripts/build_natura_raster.py

```diff
@@ -151,12 +151,7 @@ def unify_protected_shape_areas(inputs, natura_crs, out_logging):
         shp = gpd.read_file(i).to_crs(natura_crs)
         total_files += 1
         try:
-            shp.geometry = shp.apply(
-                lambda row: make_valid(row.geometry)
-                if not row.geometry.is_valid
-                else row.geometry,
-                axis=1,
-            )
+            shp.geometry = shp.geometry.make_valid()
```

Comment:
A curiosity question: what is the reason of this change? Is it linked with a fact that we transferred from using a number of shape files by continents a single global one?

Reply:
Because the make_valid() function has the same effect without the lambda function, that usually adds more overhead

Reply:
Ah, `make_valid` is vectorised! Good to know. Thanks a lot for the explanation :)

Reply:
A pleasure :)